### DISCUSSION TITLE
Deepin control center => Network is not exist after update

### DISCUSSION THREAD

**Discussion by yaaaaa** [linuxdeepin/developer-center#3388] [Q&A | 问答板块] [answered]
In both archbased distros (Garuda on PC and Manjaro on Notebook).

**Comment by BLumia** [comment, marked as answer]
Yes since the [`dde-network-core`](https://github.com/linuxdeepin/dde-network-core/) package provided the network configuration plugin for the control center app, and it's not (yet) packaged to Arch Linux due to some portability issues.

~~For a workaround, you can use the network options (provided by [`plasma-nm`](https://archlinux.org/packages/extra/x86_64/plasma-nm/) if I remembered correctly) from [`systemsettings`](https://archlinux.org/packages/extra/x86_64/systemsettings/). since they a both a front-end of `network-manager`, or use `nmtui` or other similar utils.~~ edit: should be fixed now, update the packages would help :)

**Reply by BLumia** [reply]
Update: should be fixed now by updating the packages, [deepin-network-core](https://archlinux.org/packages/community/x86_64/deepin-network-core/) will be installed as a dependency of deepin-control-center to provide the network settings :)

**Comment by Cardellino** [comment]
Hello BLumia,
I also had this problem...
With today's update, the network icon is visible again in the Control Center...
ArchLinux x86_64
Kernel 5.19.6-arch1-1
Deepin 20.6
Thank you for troubleshooting

**Reply by BLumia** [reply]
> With today's update, the network icon is visible again in the Control Center...

So, after the update it was/should be fixed :)
